src/aerialmap_display.h

```diff
@@ -143,52 +150,45 @@ protected Q_SLOTS:
       assert(!material.isNull());
     }
   };
+
+  /// the tile scene objects
   std::vector<MapObject> objects_;
 
-  ros::Subscriber coord_sub_;
+  /// the subscriber for the NavSatFix topic
+  ros::Subscriber navsat_fix_sub_;
 
   // properties
   RosTopicProperty* topic_property_;
   StringProperty* tile_url_property_;
   IntProperty* zoom_property_;
   IntProperty* blocks_property_;
-  FloatProperty* resolution_property_;
   FloatProperty* alpha_property_;
   Property* draw_under_property_;
 
+  /// the alpha value of the tile's material
   float alpha_;
+  /// determines which render queue to use
   bool draw_under_;
+  /// the URL of the tile server to use
   std::string tile_url_;
+  /// the zoom to use (Mercator)
   int zoom_;
+  /// the number of adjacent tile blocks
```

Comment:
Hm, I think this is too vague. I would say "the number of tiles loaded in each direction around the center tile"

Reply:
ok